content/en/users/cloud-compute/ec3/apps/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "EC3 applications and tools"
+type: docs
+weight: 30
+description: >
+  How to run scientific applications and tools with EC3
+---
+
+This section documents how to run some applications and use the existing tools
+with EC3.
+
+## How to run scientific applications in EC3
+
+### NAMD cluster
+
+To deploy NAMD clusters, please select one of the available LRMS (Local Resource
+Management System) and choose NAMD from the list of applications.
+
+## How to use generic tools/practices in EC3
+
+### Apache Mesos
+
+To deploy a virtual cluster with Apache Mesos as a compute cluster, please
+select Mesos from the list of available Local Resource Management System (LRMS).
+
+### Chronos
+
+To deploy a virtual cluster with Chronos as fault-tolerant scheduler, please
+select Chronos from the list of available LRMS.
+
+### ECAS cluster
+
+Check the dedicated [ECAS documentation](./ecas/).
+
+### Kubernetes
+
+Check the
+[Cloud Container Compute documentation](../../../cloud-container-compute).
+
+### Marathon
+
+To deploy a virtual cluster with Marathon as an orchestration, please select
+Chronos from the list of available LRMS.
```

Comment:
Typo?
I would have expected to have Marathon, Chronos and Mesos quite linked.

Reply:
Now it changed, the correct selection is: Mesos + Marathon + Chronos

Reply:
So instead of three entries for each of them should we have one entry only for Mesos + Marathon + Chronos?